src/test/java/emissary/util/shell/ExecutrixTest.java

```diff
@@ -24,37 +24,19 @@
 import java.util.stream.IntStream;
 
 import emissary.test.core.UnitTest;
+import org.apache.commons.lang3.SystemUtils;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 class ExecutrixTest extends UnitTest {
     private Executrix e;
-    private final boolean isWindows = System.getProperty("os.name").contains("Window");
```

Comment:
I know this is on me. But should this be reverted to match the original. Then Emissary 8 can remove all the windows references? Not aware of the difference between System.getProperty and SystemUtils.os_name

Reply:
This change is part of what was requested in the ticket. From the ticket:


"We have quite a few test classes that rely on pulling the java.io.tmpdir value from System.getProperty.

UnitTest, ExecutrixTest, ...

I think it would be a good idea to move to using the Files.createTempFile or Files.createTempDirectory where possible to abstract us away from how temporary files are created (even if that behind the scenes leverages the same call)."

Reply:
Actually, reading it again, I may have incorrectly interpreted the comment as "we should stop using System.getProperty". I can revert the change if that's preferred.